client/js/stores/snippet-store.js

```diff
@@ -11,6 +11,7 @@ class SnippetStore {
     this.bindActions(RatingActions);
     this.state = {
       snippets: [],
+      currentUserSnippets: [],
```

Comment:
ok but these state would also change depending on snippets collections so definitely they will be different for snippets and currentUserSnippets are you planning to somehow switch them depending on view?